cmd/ebrelayer/main.go

```diff
@@ -155,18 +155,27 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 	validatorMoniker := args[3]
 	mnemonic := args[4]
 
-	// Universal logger
-	logger := tmLog.NewTMLogger(tmLog.NewSyncWriter(os.Stdout))
+	logger, err := zap.NewProduction()
```

Comment:
We should replace the standard logger with the uber logger -

`zap.RedirectStdLog(sub.SugaredLogger.Desugar());
`

Having a mix of logging styles in the output isn't great.